fairlogger.sh

```diff
@@ -0,0 +1,46 @@
+package: FairLogger
+version: "%(tag_basename)s"
+tag: v1.1.0
+source: https://github.com/FairRootGroup/FairLogger
+build_requires:
+ - CMake
+ - "GCC-Toolchain:(?!osx)"
+incremental_recipe: |
+  cmake --build . --target install ${JOBS:+-- -j$JOBS}
+  mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+---
+mkdir -p $INSTALLROOT
+
+cmake $SOURCEDIR                                                 \
+      ${CXX_COMPILER:+-DCMAKE_CXX_COMPILER=$CXX_COMPILER}        \
+      ${CMAKE_BUILD_TYPE:+-DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE}  \
+      ${CXXSTD:+-DCMAKE_CXX_STANDARD=$CXXSTD}                    \
+      -DCMAKE_INSTALL_PREFIX=$INSTALLROOT                        \
+      -DDISABLE_COLOR=ON
+
+cmake --build . ${JOBS:+-- -j$JOBS}
```

Comment:
I think `make && make install` are much clearer. We are using CMake to generate Makefiles in any case. I am quite neutral about that but for the moment my feeling is that it makes the recipe less readable.

Reply:
(Actually TBH we are also thinking of using ninja so maybe leave it like this 🙂)

Reply:
So, do you want it changed or not? ;)